packages/client3/pages/artist/pool/[address]/pool-details-grid-right.tsx

```diff
@@ -15,6 +18,63 @@ type Props = {
 function PoolDetailsRightGrid({ poolData, tranchedPoolData }: Props) {
   const terms = poolData?.terms;
 
+  // Note: Need to use Borrower.sol to interact with TranchedPool.sol
+  const borrowerContract = useContract("Borrower", poolData.borrowerContract);
+
+  const goldfinchFactory = useContract(
+    "GoldfinchFactory",
+    CONTRACT_ADDRESSES.GoldfinchFactory
+  );
+
+  const disabled =
+    tranchedPoolData.estimatedTotalAssets - tranchedPoolData.totalDeployed == 0
+      ? true
+      : false;
```

Comment:
Minor thing Can use `!` operator to cast it to boolean
```suggestion
  const disabled =
    !tranchedPoolData.estimatedTotalAssets - tranchedPoolData.totalDeployed 
```

Reply:
Right, thanks!